docs/legacy/concepts/architecture.mdx

```diff
@@ -201,6 +201,7 @@ enum ErrorCode {
   MethodNotFound = -32601,
   InvalidParams = -32602,
   InternalError = -32603,
+  ElicitationRequired = -32000,
```

Comment:
-32000 conflicts with the `UnsupportedVersionError` proposed by https://github.com/modelcontextprotocol/modelcontextprotocol/issues/1442. And -32001 conflicts with `SessionRequired`.